modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/AssignQualityScore.pm

```diff
@@ -98,6 +104,30 @@ sub write_output {
     close $out_fh;
 
     $self->warning("Scores written to $output_file!");
+
+    foreach my $mlss_id ( @aln_mlss_ids ) {
+        my $mlss = $mlss_adaptor->fetch_by_dbID($mlss_id);
+        print "Writing n_${member_type}_wga_score to the database\n" if $self->debug;
+        $self->_write_n_tag($mlss, "${member_type}_wga", \%max_quality);
+        print "Tag: n_${member_type}_wga_score written!\n\n" if $self->debug;
+    }
+}
+
+sub _write_n_tag {
+    my ($self, $mlss, $label, $scores) = @_;
+
+    my %distrib_hash;
+    foreach my $score ( values %$scores ) {
+        my $floor_score = int($score/25)*25;
+        $distrib_hash{$floor_score} += 1;
+    }
+
+    my $n_tot = 0;
+    my $n_over_threshold = 0;
+    foreach my $distrib_score ( keys %distrib_hash ) {
+        my $tag = sprintf('n_%s_%s', $label, $distrib_score // 'null');
```

Comment:
I may be wrong, but isn't `$distrib_score`, a key of a hash, enforced to be defined? Maybe you meant to use `||` instead?

Reply:
As above